content/docs/applicationmobility/_index.md

```diff
@@ -7,7 +7,7 @@ Description: >
 ---
 
 {{% pageinfo color="primary" %}}
-Application Mobility is available with [APEX Navigator for Kubernetes](https://www.dell.com/en-ca/dt/apex/storage/public-cloud/navigator.htm#kubernetes)
+We are pleased to announce that Application Mobility will be available with Container Storage Modules starting early next year (2025).
```

Comment:
This message needs to be updated in the live docs as well.

Reply:
Same message should be updated in the Release notes page as well (https://dell.github.io/csm-docs/docs/applicationmobility/release/)

Reply:
Yes, I will have separate PR for 1.11.1 branch once its merged to main.